### ISSUE TITLE
It only syncs with the main virtual device

### ISSUE THREAD

**Issue by Peabuttt** [Frosthaven/voicemeeter-windows-volume#33]
Hey,

I noticed that moving windows sliders on the other virtual devices ( aux and vaio 3 ) will not affect voicemeeter sliders.

![image](https://user-images.githubusercontent.com/107125078/209175112-bfb1017c-9208-4c77-8001-c91075931cff.png)
![image](https://user-images.githubusercontent.com/107125078/209175280-594f07a6-500a-4b18-8e21-04cb3e11e48c.png)

And i can confirm volumes do not change.

**Comment by Frosthaven** [comment]
At the moment, this applet is designed to specifically tie only the default windows audio device (the one being used by the systray) to 1 or more voicemeeter channels.

I'm not against something to tie in the others, but for now it's beyond the scope of what this applet offers. Happy to take pull requests :)

**Comment by Peabuttt** [comment]
Unfortunately i don't have the knowledge to develop this feature. It would be very handy to me because i use a deej (a diy mixer) and i really want to control several voicemeter sliders.

Thanks for the feedback.

**Comment by Frosthaven** [comment]
Right, so let me make sure I've fully captured your request in the off chance I stumble into some free time. Correct me if I'm wrong here!

If we're looking at Windows' `Settings > System > Sound` panel, you are wanting to be able to map each of the devices listed under input/output to different Voicemeeter channels. Does that seem like a good brief description?

**Comment by Frosthaven** [comment]
I'm going to re-open this so that it stays as a standing request instead of flat out refusing to consider it :)

**Comment by Peabuttt** [comment]
First of all thank you for reopening and considering my request.

Yes you are correct, although i don't need to match all of them. I'm currently using 3 sliders for output and 3 for input. Deej software reads physical sliders/knobs connected to an Arduino and matches those values with audio devices or software volume. So i want to sync voicemeter potato 3 outputs (1 is already synced with your software) and inputs 1 to 3 to windows audio devices, just like we do for master audio slider.

To understand even better. For the outputs i have general volume, comms and media (spotify, mpc...). For inputs i have headset mic, desk mic and capture card sound. Currently i can only control general volume with my mixer, for the others i need to move the slider inside voicemeter.
